### PR TITLE
Release v0.38.0

### PR DESCRIPTION
**Pull request by sergio-mena**
[Rendered CHANGELOG.md](https://github.com/cometbft/cometbft/blob/release/v0.38.0/CHANGELOG.md)

E2E manual run on this branch: https://github.com/cometbft/cometbft/actions/runs/6158591090

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Review by hvanz — APPROVED** [review]
The following entry in the changelog needs some formatting; otherwise, LGTM.

> `[mempool/clist_mempool]` \#890 Prevent a transaction to appear twice in the mempool  (@otrack)

**Comment by hvanz** [comment]
The following two entries could be combined into one:
> [abci] Added cli commands for PrepareProposal and ProcessProposal. (https://github.com/tendermint/tendermint/pull/8656)
> [abci] Added cli commands for PrepareProposal and ProcessProposal. (https://github.com/tendermint/tendermint/pull/8901)

**Comment by sergio-mena** [comment]
> The following two entries could be combined into one

I agree with you, but this is from `v0.37.0`, so I'd leave it as it is

**Review by jmalicevic — APPROVED** [review]

File: .changelog/v0.38.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+*September 12, 2023*
+
+This release includes the second part of ABCI++, called ABCI 2.0.
+ABCI 2.0 introduces ABCI methods `ExtendVote` and `VerifyVoteExtension`.
+These new metthods allow the application to add data (opaque to CometBFT),
```
Comment (thanethomson) [inline .changelog/v0.38.0/summary.md:5]:
```suggestion
These new methods allow the application to add data (opaque to CometBFT),
```

Applying this change will require rebuilding the changelog with unclog.

**Review by thanethomson — APPROVED** [review]
Nice! Just 1 minor spelling nit, but otherwise LGTM!

**Pull request merged**